### ISSUE TITLE
please provide a phrase table demo

### ISSUE THREAD

**Issue by keyboardWitch** [machinalis/yalign#11]
Hi,
  I found that this align tool is very useful. And I wanna to train a model of my own, but I do not have any phrase table could you provide a phrase table demo?
many thanks!

**Comment by echan00** [comment]
Same here, looking for a Chinese dictionary (phrase table)

**Comment by echan00** [comment]
Did you have any luck?

**Comment by keyboardWitch** [comment]
I turned to an other align program.  Based on gale church align and the bleu score of machine translation

**Comment by echan00** [comment]
Was it bleualign? Or something else? It would be great if you can share what you used.

**Comment by keyboardWitch** [comment]
Yes bleualign. I made a task queue to auto align the parellel web pages downloaded by scrapy. The machine translation is from google .

**Comment by echan00** [comment]
Thanks!   I was initially also using bleualign, but had too many documents to align and using google translate is too expensive for my project.

**Comment by keyboardWitch** [comment]
You can use the free google translation , slow down the request and increase the number of concurrent
the yalign need language models. I think bleualign is more useful for short pages alignment

**Comment by LukeTu** [comment]
Same problem here. But I saw someone already has their language pair for alignment.
I also found a parper that talked about improving the performance of Yalign. 
https://arxiv.org/abs/1512.01641 
didn't mentioned the creation of dictionary(phrase table), but might be helpful.
